### ISSUE TITLE
tag_owner_organisation requested in get_transmitters

### ISSUE THREAD

**Issue by LienReyserhove** [inbo/etn#69]
In the data retreived by the function `get_transmitters()`, the column `tag_owner_organisation`, present in the [ETN view](http://www.lifewatch.be/etn/tags) for tags, is not included.
I would like to have this information included, is it possible to integrate this column?

**Comment by damianooldoni** [comment]
Yes, indeed. I see that in `vliz.tags` there is a column called `owner_group_fk`, which is the primary key of table `vliz.etn_group`. 
Adding a `LEFT JOIN` to [transmitters_query #L51](https://github.com/inbo/etn/blob/master/R/get_transmitters.R#L51) should solve it.

In other words, this: 
```r
transmitters_query <- glue_sql("
      SELECT tags.*, animals.projectcode
      FROM vliz.tags
        LEFT JOIN vliz.animal_tag_release ON (animal_tag_release.tag_fk = tags.id_pk)
        LEFT JOIN vliz.animals_view animals ON (animals.id_pk = animal_tag_release.animal_fk)
      WHERE projectcode IN ({project*})",
    project = animal_project,
    .con = connection
  )
```
should become this:
```r
tags_with_owners_query <- glue_sql("
      SELECT tags.*, animals.projectcode, etn_group.name as owner_organization
      FROM vliz.tags
        LEFT JOIN vliz.animal_tag_release ON (animal_tag_release.tag_fk = tags.id_pk)
        LEFT JOIN vliz.animals_view animals ON (animals.id_pk = animal_tag_release.animal_fk)
        LEFT JOIN vliz.etn_group  ON (tags.owner_group_fk = etn_group.id_pk)
      WHERE projectcode IN ({project*})",
                               project = animal_project,
                               .con = my_con
)
```
This should add column `owner_organization` to the output of function `get_transmitters`.
@stijnvanhoey : may I add it via branch and send PR?

**Comment by stijnvanhoey** [comment]
sure! Maybe use `tag_owner_organisation` as column name to have the similarity with the app?

**Comment by damianooldoni** [comment]
Thanks @stijnvanhoey.
I was thinking to use `owner_organization` as there are already two columns starting with `owner_`:
1. `owner_pi`
2. `owner_group_fk`. 

I found thus a natural choice to use `owner_organization`. Still, you have the last word...

**Comment by peterdesmet** [comment]
@damianooldoni this can be implemented: add `owner_organization` to `my_animals`

**Comment by damianooldoni** [comment]
Hi @LienReyserhove : I see you closed this issue: that means that PR #70 can be merged? I addded you as reviewer so you can test and approve (?) the changes. Thanks.

**Comment by peterdesmet** [comment]
This is now included in `get_tags()` (but not via PR #70):

https://github.com/inbo/etn/blob/aa0b739f3866c3b83126fb275846cd921c724c6d/tests/testthat/test-get_tags.R#L30
